templates/index.html

```diff
@@ -1,7 +1,14 @@
 {% extends "base.html" %}
 
 {% block content %}
-Hello World!!!
+	<div class="inner cover">
```

Comment:
This could also go in base as it's the page's header which will be shown on every other page :)